lib/services.identity.js

```diff
@@ -669,37 +669,9 @@ function updateIdentity(req, res, next) {
       brIdentity.setRoles(req.user.identity, identity, callback);
     }],
     update: ['updateRoles', function(callback, results) {
-      var identity = {id: results.getId};
-
-      // conditional values only set if preset
-      if(req.body.description) {
-        identity.description = req.body.description;
-      }
-      if(req.body.email) {
```

Comment:
hmm, it looks like it was always getting dropped when passed down to brIdentity.update() https://github.com/digitalbazaar/bedrock-identity/pull/11/files#diff-ed92430a17feb1ffc2f12515a297bc2dL239

I'll add it in, looks like it is supposed to be there.